modules/swagger-codegen/src/main/resources/Java/ApiClient.mustache

```diff
@@ -1,8 +1,9 @@
 package {{invokerPackage}};
 
 import com.fasterxml.jackson.annotation.*;
-import com.fasterxml.jackson.databind.*;
-import com.fasterxml.jackson.datatype.joda.*;
+import com.fasterxml.jackson.databind.*;{{#java8}}
+import com.fasterxml.jackson.datatype.jsr310.*;{{/java8}}{{^java8}}
+import com.fasterxml.jackson.datatype.joda.*;{{/java8}}
```

Comment:
That's my fault, sorry ! I didn't know about new line trimming so I put a wrong comment :sweat_smile: ...